qa/workunits/ceph-helpers.sh

```diff
@@ -17,6 +17,15 @@
 # MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
 # GNU Library Public License for more details.
 #
+if [ x"$SRC_PATH"x = xx ]; then
+  CEPH_DIR=`dirname $0`
+  CEPH_DIR=`(cd $CEPH_DIR/../..; pwd)`
+fi
+
+CEPH_SRC=$CEPH_DIR/src
+CEPH_BIN=$CEPH_SRC
```

Comment:
@liewegas @tchaikov 
How desperate are we that the scripts can still be executed standalone?

I can construct `CEPH_{ROOT,BIN,LIB}` relatively easy for standalone, cmake and autobuild,
(updated the commit to match that)
but getting PYTHONPATH requires (in Cmake env):
    `./build/lib/cython_modules/lib.freebsd-11.0-CURRENT-amd64-2.7/`
to find `rados.so` 
Which is sort a far fetched path. 

I see a few solutions, but one relatively nice one would be to Cmake to actually install rados.so into something like `./build/lib` or `./build/lib/pybind`  OR replace `lib` with `bin`